com.unity.multiplayer.mlapi/Editor/MLAPIProfiler.cs

```diff
@@ -40,36 +40,38 @@ class ProfilerContainer
             public ProfilerTick[] ticks;
 
             public byte[] ToBytes()
-			{
-				BitStream stream = new BitStream();
-				BitWriter writer = new BitWriter(stream);
-				writer.WriteUInt16Packed((ushort)ticks.Length);
-
-				for (int i = 0; i < ticks.Length; i++)
-				{
-					ticks[i].SerializeToStream(stream);
-				}
-
-				return stream.ToArray();
-			}
-
-			public static ProfilerContainer FromBytes(byte[] bytes)
-			{
-				ProfilerContainer container = new ProfilerContainer();
-				BitStream stream = new BitStream(bytes);
-				BitReader reader = new BitReader(stream);
-				ushort count = reader.ReadUInt16Packed();
-				container.ticks = new ProfilerTick[count];
+            {
+                BitStream stream = new BitStream();
+                BitWriter writer = new BitWriter(stream);
+                writer.WriteUInt16Packed((ushort)ticks.Length);
+
+                for (int i = 0; i < ticks.Length; i++)
+                {
+                    ticks[i].SerializeToStream(stream);
+                }
+
+                return stream.ToArray();
+            }
+
+            public static ProfilerContainer FromBytes(byte[] bytes)
+            {
+                MLAPI.NetworkingManager manager = NetworkingManagerEditor.GetAnyNetworkingManager();
+
+                ProfilerContainer container = new ProfilerContainer();
+                BitStream stream = new BitStream(bytes);
+                BitReader reader = new BitReader(manager, stream);
+                ushort count = reader.ReadUInt16Packed();
+                container.ticks = new ProfilerTick[count];
                 for (int i = 0; i < count; i++)
-				{
-					container.ticks[i] = ProfilerTick.FromStream(stream);
-				}
+                {
+                    container.ticks[i] = ProfilerTick.FromStream(manager, stream);
```

Comment:
Same story here, in the FromStream lambda, you can first read the NM id, then de-ref and not need to pass this in.

Reply:
Also was able to break this dep same was as above